bedrock/firefox/templates/firefox/all.html

```diff
@@ -33,6 +35,19 @@
   {% endif %}
 {% endblock %}
 
+{% block site_header_share %}
+  {% if channel == 'release' -%}
+    {% set share_urls = { 'twitter': 'http://mzl.la/1IMdR5T', 'googleplus': 'http://mzl.la/1y7TgOf', 'facebook': 'http://mzl.la/1ATF965' } -%}
+    {% if l10n_has_tag('share_cta') -%}
+      {% set share_text = _('Thanks to contributions of Mozilla community members around the world, Firefox is available in %s languages:')|format(num_languages) %}
```

Comment:
This is not going to work for a lot of locales (languages need to be adapted to the actual numbers).

We had the same issue in the past, and ended up using "in more than X languages" (not sure what X is these days).